packages/flutter_localizations/lib/src/l10n/material_km.arb

```diff
@@ -0,0 +1,54 @@
+{
+  "scriptCategory": "dense",
+  "timeOfDayFormat": "h:mm a",
+  "openAppDrawerTooltip": "បើកម៉ឺនុយរុករក",
+  "backButtonTooltip": "ថយក្រោយ",
+  "closeButtonTooltip": "បិទ",
+  "deleteButtonTooltip": "យកចេញ",
+  "nextMonthTooltip": "ខែក្រោយ",
+  "previousMonthTooltip": "កាលពីខែមុន",
+  "nextPageTooltip": "ទំព័របន្ទាប់",
+  "previousPageTooltip": "ទំព័រមុន",
+  "showMenuTooltip": "បង្ហាញម៉ឺនុយ",
+  "aboutListTileTitle": "អំពី $applicationName",
+  "licensesPageTitle": "អាជ្ញាប័ណ្ណ",
+  "pageRowsInfoTitle": "$firstRow-$lastRow ពី $rowCount",
+  "pageRowsInfoTitleApproximate": "$firstRow–$lastRow ពីប្រហែល $rowCount",
+  "rowsPerPageTitle": "ជួរមុខ:",
+  "tabLabel": "ថេប $tabIndex ពី $tabCount",
+  "selectedRowCountTitleOne": "បានជ្រើស 1 ធាតុ",
+  "selectedRowCountTitleOther": "បានជ្រើស $selectedRowCount ធាតុ",
+  "cancelButtonLabel": "បោះបង់",
+  "closeButtonLabel": "បិទ",
+  "continueButtonLabel": "បន្ត",
+  "copyButtonLabel": "ចម្លង",
+  "cutButtonLabel": "កាត់",
+  "okButtonLabel": "យល់ព្រម",
+  "pasteButtonLabel": "ដាក់",
+  "selectAllButtonLabel": "ជ្រើសទាំងអស់",
+  "viewLicensesButtonLabel": "មើលអាជ្ញាប័ណ្ណ",
+  "anteMeridiemAbbreviation": "AM",
+  "postMeridiemAbbreviation": "PM",
+  "timePickerHourModeAnnouncement": "ជ្រើសរើសមួយម៉ោង",
+  "timePickerMinuteModeAnnouncement": "ជ្រើសនាទី",
+  "signedInLabel": "ចូល",
+  "hideAccountsLabel": "លាក់គណនី",
+  "showAccountsLabel": "បង្ហាញគណនី",
+  "modalBarrierDismissLabel": "បិទ",
+  "drawerLabel": "ម៉ឺនុយរុករក",
+  "popupMenuLabel": "ម៉ឺនុយលេចឡើង",
+  "dialogLabel": "ប្រអប់",
+  "alertDialogLabel": "ការជូនដំណឹង",
+  "searchFieldLabel": "ស្វែងរក",
+  "reorderItemToStart": "TBD",
```

Comment:
Is it okay to have these placeholder translations? Or should they fall through to e.g. english so the UI shows something at least somewhat useful?

Reply:
This is the pattern we've been using for placeholders. We should have real translations in a week or two.